README.md

```diff
@@ -6,7 +6,9 @@ MLC LLM Performance Benchmarking
 | Model      | GPU         | MLC LLM (tok/sec) | Exllama (tok/sec) |
 |------------|-------------|-------------------|-------------------|
 | Llama2-7B  | RTX 3090 Ti | 154.1             | 116.38            |
+| Llama2-7b  | RTX 4090    | 167.6             | 123.35            |
```

Comment:
On 4090, with the latest MLC HEAD + prebuilt TVM Unity, I got the numbers below for MLC:
- 7B: 184.9 token/sec
- 13B: 106.4 token/sec